html-block-and-inline/html-block-and-inline/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+  </head>
+  <body>
+    <h1><b>Blue Americana Snuggie &reg; Blanket</b></h1>
```

Comment:
You won't need the `b` tags in this situation as the `h` tags are bold by default.